Resources/Audio/_Harmony/Weapons/Guns/Gunshots/attributions.yml

```
@@ -12,3 +12,9 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from cmss13"
   source: "https://github.com/cmss13-devs/cmss13/blob/8086109b2dbf8d1d98aec106a07d86b65fe63e1b/sound/weapons/gun_sniper.ogg"
+
+- files: ["dmr_fire1.ogg", "dmr_fire2.ogg", "dmr_fire3.ogg"]
+  license: "Custom"
+  copyright: "Halo © Microsoft Corporation. This audio file was created under Microsoft's Game Content Usage Rules using assets from Halo:Reach, and it is not endorsed by or affiliated with Microsoft. More info here: https://www.xbox.com/en-us/developers/rules. Pulled from Halo:Reach, made by Bungie"
+  source: "https://youtu.be/0SEE9n-ciUc"
+  
```

The last line needs to be empty
```suggestion
  source: "https://youtu.be/0SEE9n-ciUc"

```